sdks/js/packages/core/react/components/organization/plans/pricing-column.tsx

```diff
@@ -0,0 +1,354 @@
+import { useNavigate } from '@tanstack/react-router';
+import { useFrontier } from '~/react/contexts/FrontierContext';
+import dayjs from 'dayjs';
+import { toast } from 'sonner';
+import * as _ from 'lodash';
+import { memo, useCallback, useEffect, useMemo, useState } from 'react';
+
+import { Button, Flex, Text, ToggleGroup, Image } from '@raystack/apsara';
+import {
+  IntervalKeys,
+  IntervalLabelMap,
+  IntervalPricingWithPlan,
+  PlanIntervalPricing
+} from '~/src/types';
+import { usePlans } from './hooks/usePlans';
+import { PlanChangeAction, getPlanChangeAction } from '~/react/utils';
+import {
+  DEFAULT_DATE_FORMAT,
+  DEFAULT_DATE_SHORT_FORMAT
+} from '~/react/utils/constants';
+import checkCircle from '~/react/assets/check-circle.svg';
+import Amount from '~/react/components/helpers/Amount';
+
+import plansStyles from './plans.module.css';
+import Skeleton from 'react-loading-skeleton';
+
+interface FeaturesListProps {
+  features: string[];
+  plan: IntervalPricingWithPlan;
+}
+
+const FeaturesList = ({ features, plan }: FeaturesListProps) => {
+  return features.map(feature => {
+    const planFeature = _.get(plan.features, feature, {
+      metadata: {}
+    });
+    const productMetaDataFeatureValues = plan.productNames
+      .map(name => _.get(planFeature.metadata, name))
+      .filter(value => value !== undefined);
+    // piciking the first value for feature metadata, in case of muliple products in a plan, there can be multiple metadata values.
```

Comment:
```suggestion
    // picking the first value for feature metadata, in case of multiple products in a plan, there can be multiple metadata values.
```